<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>txt_data_load_dlg</class>
 <widget class="QDialog" name="txt_data_load_dlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TextDataLoad</string>
  </property>
  <widget class="QGroupBox" name="regex">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>271</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Regular expression</string>
   </property>
   <widget class="QLineEdit" name="regexDelim">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="regexNA">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="regexComment">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Delim</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>NA</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Comment</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="SquareTest">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Data</string>
   </property>
   <widget class="QCheckBox" name="cbForceAlign">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Force columns aligned</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Import as</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbImportAs">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>scatter series</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>sampled1d</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>sampled2d</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>scatter1d</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>scatter2d</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Layout">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>241</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Layout</string>
   </property>
   <widget class="QRadioButton" name="rbRowMajor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Row major</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbColMajor">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Column major</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Start row</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Start column</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leStartRow">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leStartCol">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="SpecialTokens">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>241</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Special tokens</string>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Illegal token as</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbIllegalAs">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentText">
     <string>fail</string>
    </property>
    <item>
     <property name="text">
      <string>fail</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ignore</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>empty</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>nan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>zero</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>emptyl token as</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbEmptyAs">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>skip</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>nan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>zero</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="dbBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>280</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbStatus">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btReload">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>280</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Reload</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dbBox</sender>
   <signal>accepted()</signal>
   <receiver>txt_data_load_dlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>495</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dbBox</sender>
   <signal>rejected()</signal>
   <receiver>txt_data_load_dlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>495</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
